styles.css

```
@@ -0,0 +1,290 @@
+*{
+  font-family: 'GTWalsheim','Didact Gothic' , sans-serif;
+}
+
+body{
+  margin: 0;
+}
+
+/*--------------------------------seccion NAV---------------------------------*/
+
+
+.mainNav{
+  display: flex;
+  justify-content: space-between;
+  margin: 26px 32px;
+}
+
+.closed.menu-items-toggle, .closed .home-logo,.closed .phone-link img{
```

Avoid using tag names for styling. If possible, use a class. Also, if you have more than a rule you can write each one in a line to have good code readability.
```suggestion
.closed.menu-items-toggle,
.closed .home-logo,
.closed .phone-link-img {
```